homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,19 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+	if(command == "add"){
+		return std::to_string(first+second);
+	}else if(command == "subtract"){
+		return std::to_string(first-second);
+	}else if(command == "multiply"){
+		return std::to_string(first*second);
+	}else if(command == "divide"){
+		if(second == 0){
+			return "Division by 0";
+		}else{
+			return std::to_string(first/second);
+		}
+	}else{
```

Comment:
Add space between operations like '+' '-' '!=' '==' etc... and also after `if` `else if` `for` etc..

If you return variable here, you don't need to use `else if`